CHANGELOG.md

```diff
@@ -4,6 +4,26 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.10.2](https://github.com/go-gitea/gitea/releases/tag/v1.10.2) - 2020-01-02
+* BUGFIXES
+  * Allow only specific Colums to be updated on Issue via (#9539) (#9580)
```

Comment:
```suggestion
  * Allow only specific Colums to be updated on Issue via API (#9539) (#9580)
```

Reply:
@6543 You added the via at the end of this so in that case API should be added at end too